[BUG]Ferroamp battery size setting

When I change #define TOTAL_CELL_AMOUNT in FERROAMP-CAN.cpp to a value over 120 I get a compile error:

_C:\Arduino15\Arduino\Battery-Emulator-8.12.0\Battery-Emulator-8.12.0\Software\src\inverter\FERROAMP-CAN.cpp:14:27: error: narrowing conversion of '480' from 'int' to 'uint8_t' {aka 'unsigned char'} [-Wnarrowing]
   14 | #define TOTAL_CELL_AMOUNT 480  //Adjust this parameter in steps of 120 to add another 14,2kWh of capacity
      |                           ^~~
C:\Arduino15\Arduino\Battery-Emulator-8.12.0\Battery-Emulator-8.12.0\Software\src\inverter\FERROAMP-CAN.cpp:36:34: note: in expansion of macro 'TOTAL_CELL_AMOUNT'
   36 |                         .data = {TOTAL_CELL_AMOUNT, (uint8_t)(TOTAL_CELL_AMOUNT >> 8), MODULES_IN_SERIES,
      |                                  ^~~~~~~~~~~~~~~~~
C:\Arduino15\Arduino\Battery-Emulator-8.12.0\Battery-Emulator-8.12.0\Software\src\inverter\FERROAMP-CAN.cpp:14:27: error: narrowing conversion of '480' from 'int' to 'uint8_t' {aka 'unsigned char'} [-Wnarrowing]
   14 | #define TOTAL_CELL_AMOUNT 480  //Adjust this parameter in steps of 120 to add another 14,2kWh of capacity
      |                           ^~~
C:\Arduino15\Arduino\Battery-Emulator-8.12.0\Battery-Emulator-8.12.0\Software\src\inverter\FERROAMP-CAN.cpp:43:34: note: in expansion of macro 'TOTAL_CELL_AMOUNT'
   43 |                         .data = {TOTAL_CELL_AMOUNT, (uint8_t)(TOTAL_CELL_AMOUNT >> 8), MODULES_IN_SERIES,
      |                                  ^~~~~~~~~~~~~~~~~
exit status 1

Compilation error: narrowing conversion of '480' from 'int' to 'uint8_t' {aka 'unsigned char'} [-Wnarrowing]_



- Software version: 8.12.0
- Battery used: [KIA_HYUNDAI_64_BATTERY]
- HW_LILYGO

--- a/Software/src/inverter/FERROAMP-CAN.h
+++ b/Software/src/inverter/FERROAMP-CAN.h
@@ -49,14 +49,14 @@ Change the following only if your inverter is generating fault codes about volta
                           .ext_ID = true,
                           .DLC = 8,
                           .ID = 0x7320,
-                          .data = {TOTAL_CELL_AMOUNT, (uint8_t)(TOTAL_CELL_AMOUNT >> 8), MODULES_IN_SERIES,
+                          .data = {(TOTAL_CELL_AMOUNT & 0xFF), (uint8_t)(TOTAL_CELL_AMOUNT >> 8), MODULES_IN_SERIES,
                                    CELLS_PER_MODULE, (uint8_t)(VOLTAGE_LEVEL & 0x00FF), (uint8_t)(VOLTAGE_LEVEL >> 8),
                                    AH_CAPACITY, (uint8_t)(AH_CAPACITY >> 8)}};
   CAN_frame PYLON_7321 = {.FD = false,
                           .ext_ID = true,
                           .DLC = 8,
                           .ID = 0x7321,
-                          .data = {TOTAL_CELL_AMOUNT, (uint8_t)(TOTAL_CELL_AMOUNT >> 8), MODULES_IN_SERIES,
+                          .data = {(TOTAL_CELL_AMOUNT & 0xFF), (uint8_t)(TOTAL_CELL_AMOUNT >> 8), MODULES_IN_SERIES,
                                    CELLS_PER_MODULE, (uint8_t)(VOLTAGE_LEVEL & 0x00FF), (uint8_t)(VOLTAGE_LEVEL >> 8),
                                    AH_CAPACITY, (uint8_t)(AH_CAPACITY >> 8)}};
   CAN_frame PYLON_4210 = {.FD = false,

